charts/secrets-store-csi-driver-provider-aws/values.yaml

```diff
@@ -1,7 +1,7 @@
 image:
   repository: public.ecr.aws/aws-secrets-manager/secrets-store-csi-driver-provider-aws
   pullPolicy: IfNotPresent
-  tag: 1.0.r2-72-gfb78a36-2024.05.29.23.03
+  tag: 0.3.9-st
```

Comment:
This tag corresponds to the docker image version id, we push this image and update the chart information accordingly. Can you remove this from your diff?

Reply:
@simonmarty removed `tag` and `.DS_STORE`